tbfparser/src/lib.rs

```diff
@@ -359,7 +356,7 @@ impl TbfHeader {
             // Header increases by the TLV and name length.
             header_length += mem::size_of::<TbfHeaderTlv>() + package_name.len();
             // How much padding is needed to ensure we are aligned to 4?
-            let pad = amount_alignment_needed(header_length as u32, 4);
+            let pad = util::amount_alignment_needed(header_length as u32, 4);
```

Comment:
```suggestion
            let pad = amount_alignment_needed(header_length as u32, 4);
```

Add `use util::amount_alignment_needed` at the top of the file.

Reply:
Done